format/format.go

```diff
@@ -313,10 +313,11 @@ func (f *fumpter) lineEnd(line int) token.Pos {
 //   //extern      | C function declarations for gccgo
 //   //sys(nb)?    | syscall function wrapper prototypes
 //   //nolint      | nolint directive for golangci
+//   //noinspection| noinspection directive for GoLand and friends
```

Comment:
add a space to this table on the left of `|`, for consistency

Reply:
Sure, means changing all those lines though but...

Reply:
that's fine :)